src/models/stores/document-group.ts

```diff
@@ -0,0 +1,137 @@
+import { FC, SVGProps } from "react";
+import { IDocumentMetadata } from "functions/src/shared";
+import { ISortedDocumentsStores, TagWithDocs } from "./sorted-documents";
+import { makeAutoObservable } from "mobx";
+import {
+  createDocMapByBookmarks,
+  createDocMapByGroups,
+  createDocMapByNames,
+  createTileTypeToDocumentsMap,
+  getTagsWithDocs,
+  DocumentCollection,
+  sortGroupSectionLabels,
+  sortNameSectionLabels
+} from "../../utilities/sort-document-utils";
+import { getTileContentInfo } from "../tiles/tile-content-info";
+import { getTileComponentInfo } from "../tiles/tile-component-info";
+import { SecondarySortType } from "./ui-types";
+
+import SparrowHeaderIcon from "../../assets/icons/sort-by-tools/sparrow-id.svg";
+
+interface IDocumentGroup {
+  icon?:FC<SVGProps<SVGSVGElement>>;
+  label: string;
+  metaDataDocs: IDocumentMetadata[];
+  stores: ISortedDocumentsStores;
+}
+
+export class DocumentGroup {
```

Comment:
A jsdoc comment explaining what a DocumentGroup is would be good, to distinguish this from other similar-named objects.